new/LIT7076QeneBeesi.xml

```diff
@@ -0,0 +1,83 @@
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Qǝne</foreign> without type indication in the manuscript. It has eight lines,
+                    suggesting that it might be of the mawaddǝs-type. 
+                    It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                    in the manuscript.</p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                    <term key="Mawaddes" cert="high"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-09-27">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7076QeneBeesi" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7076QeneBeesi" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="gez">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 34r, 35r. Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1">ብእሲ፡ ኢይጻእ፡ እምአንቀጸ፡ ቤቱ፡ እስመ፡ አረጋዊ፡ ውእቱ፡ ዘወጽአ፡ እምነ፡ ገነት፡ ንድቅ፤</l>
+                    <l n="2">መተረ፡ ቈጸለ፡ ተስፋ፡ ኪሩቤል፡ አእጹቅ<choice><orig>፡</orig><corr>፤</corr></choice></l>
+                    <l n="3">በምክረ፡ ከይሲ፡ ወሔዋን፡ ደቂቅ፤</l>
```

Comment:
Now that I see images too, I check: it seems በምክሩ፡ :-)

Reply:
I looked again and I think it's ረ. Compare the ረ in አረጋዊ and መተረ, and the ሩ in ኪሩቤል

Also, ረ is what makes sense grammatically :)